__tests__/setup-beam.test.js

```diff
@@ -16,6 +16,8 @@ async function all() {
   await testOTPVersions()
   await testElixirVersions()
   await testRebar3Versions()
+
+  await testGetVersionFromSpec()
```

Comment:
Mind you none of the other tests change, so this should give us some confidence in backward compatibility.

Reply:
Well, some actually change, but the update only made the previous implementation issue (not getting the latest latest) into evidence, so I guess this is good.